detekt-baseline.xml

```diff
@@ -7,6 +7,7 @@
     <ID>FunctionNaming:JPASpecificationDSL.kt$fun &lt;T, R : Any&gt; KProperty1&lt;T, R?&gt;.`in`(values: Collection&lt;R&gt;): Specification&lt;T&gt;</ID>
     <ID>LargeClass:LicenceController.kt$LicenceController</ID>
     <ID>LargeClass:LicenceService.kt$LicenceService</ID>
+    <ID>LongMethod:ComService.kt$ComService$fun searchForOffenderOnStaffCaseload(body: ProbationUserSearchRequest): ProbationSearchResult</ID>
```

Comment:
This is providing some good advice, worth thinking of ways to address it rather than add an exclusion

Reply:
Based on splitting out logic into local extension functions, no longer an issue